gobblin-scheduler/src/main/java/gobblin/scheduler/SchedulerDaemon.java

```diff
@@ -72,20 +72,22 @@ public SchedulerDaemon(Properties defaultProperties, Properties customProperties
     List<Service> services = Lists.<Service>newArrayList(new JobScheduler(properties));
     boolean jobExecInfoServerEnabled = Boolean
         .valueOf(properties.getProperty(ConfigurationKeys.JOB_EXECINFO_SERVER_ENABLED_KEY, Boolean.FALSE.toString()));
+    boolean adminUiServerEnabled = Boolean
+        .valueOf(properties.getProperty(ConfigurationKeys.ADMIN_SERVER_ENABLED_KEY, Boolean.FALSE.toString()));
     if (jobExecInfoServerEnabled) {
+      LOG.info("Starting the job execution info server since it is enabled");
       JobExecutionInfoServer executionInfoServer = new JobExecutionInfoServer(properties);
       services.add(executionInfoServer);
-      if (shouldRunAdminServer(properties)) {
+      if (adminUiServerEnabled) {
```

Comment:
It seemed less consistent with how the job execution server property was being done.  If you feel strongly, the method can be added back, but I would like to still set the variable as it means we don't call the method twice.